components/common/BigButton.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import styled from 'styled-components';
+
+interface ButtonProps {
+  text: string;
+  disabled: boolean;
```

Comment:
```suggestion
  isDisabled: boolean;
```

boolean 자료형이므로 `is+disabled`로 사용하는 건 어떨까요!?

Reply:
수정해볼게!!